spec/unit/installer/xcode/pods_project_generator/target_installer_spec.rb

```diff
@@ -54,7 +54,7 @@ class PodsProjectGenerator
             @pod_target.stubs(:uses_swift?).returns(true)
             @installer.send(:add_target)
             @installer.send(:native_target).resolved_build_setting('SWIFT_OPTIMIZATION_LEVEL').should == {
-              'Release' => nil,
+              'Release' => '-Owholemodule',
```

Comment:
Ah yeah looks like that was introduced in https://github.com/CocoaPods/Xcodeproj/pull/468

I'm not sure if that is a good default with us working across multiple Xcode versions, and wmo being super slow (or broken) on older Xcodes